core/src/nucleus/mod.rs

```diff
@@ -480,6 +485,7 @@ mod tests {
     }
 
     #[test]
+    /// smoke test the init of a nucleus reduction
```

Comment:
@zippy "where there's smoke, there's fire" - it just means the test doesn't do anything thoughtful that would pick up subtle implementation details, but it might pick up a blatant fatal caused by the fact that something exists/inits/constructs at all